pandas/core/dtypes/common.py

```diff
@@ -128,7 +128,7 @@ def ensure_str(value: Union[bytes, Any]) -> str:
     return value
 
 
-def ensure_int_or_float(arr: ArrayLike, copy: bool = False) -> np.ndarray:
+def ensure_int_or_float(arr: ArrayLike, copy: bool = False) -> ArrayLike:
```

Comment:
why is this changed?

Reply:
Forgot to undo that change when working back from the other PR.

Reply:
Fixed in commit.